firefox-ios/Client/Coordinators/EnhancedTrackingProtectionCoordinator.swift

```diff
@@ -13,11 +13,15 @@ protocol EnhancedTrackingProtectionCoordinatorDelegate: AnyObject {
 }
 
 class EnhancedTrackingProtectionCoordinator: BaseCoordinator,
-                                             EnhancedTrackingProtectionMenuDelegate {
+                                             TrackingProtectionMenuDelegate,
+                                             EnhancedTrackingProtectionMenuDelegate,
+                                             FeatureFlaggable {
     private let profile: Profile
     private let tabManager: TabManager
-    private let enhancedTrackingProtectionMenuVC: EnhancedTrackingProtectionMenuVC
+    private let oldEnhancedTrackingProtectionMenuVC: EnhancedTrackingProtectionMenuVC
```

Comment:
EnhancedTrackingProtectionMenuVC should be renamed to something more verbose if possible.
I do understand its old codebase but the lease we should change is `oldEnhancedTrackingProtectionMenuVC` to `legacyEnhancedTrackingProtectionMenuViewController`or `legacyETPMenuViewController`